src/bin/pg_autoctl/keeper.c

```diff
@@ -927,8 +930,17 @@ keeper_ensure_configuration(Keeper *keeper, bool postgresNotRunningIsOk)
 			return false;
```

Comment:
this error and the two above can leak `currentConfContents`

Reply:
Good point, fixed.